xlators/mgmt/glusterd/src/glusterd-snapshot-utils.c

```diff
@@ -233,14 +233,17 @@ glusterd_snap_volinfo_restore(dict_t *dict, dict_t *rsp_dict,
             goto out;
         }
 
-        /* Fetch values if present in dict These values won't
+        /* Fetch values if present in dict. These values won't
          * be present in case of a missed restore. In that case
          * it's fine to use the local node's value
          */
         if (retain_origin_path) {
             snprintf(key, sizeof(key), "snap%d.brick%d.origin_path", volcount,
                      brick_count);
             ret = dict_get_str(dict, key, &value);
+            if (!ret)
+                gf_strncpy(new_brickinfo->origin_path, value,
```

Comment:
set `new_brickinfo->path` here instead of origin_path

Reply:
Done.